spec/rubocop/directive_comment_spec.rb

```diff
@@ -198,4 +198,20 @@
       end
     end
   end
+
+  describe '#disabled_next_line?' do
+    subject { directive_comment.disabled_next_line? }
+
+    [
+      ['when disabled next line', '# rubocop:disable-next-line all', true],
+      ['when disabled current line', 'def foo # rubocop:disable all', false],
+      ['when disabled few lines', '# rubocop:disable Foo/Bar', false]
+    ].each do |example|
+      context example[0] do
+        let(:text) { example[1] }
+
+        it { is_expected.to eq example[2] }
+      end
+    end
```

Comment:
This could be replaced by a shared example for easier reading.

Reply:
Replaced